zephyr/include/rtos/interrupt.h

```diff
@@ -52,7 +52,8 @@ static inline void interrupt_unregister(uint32_t irq, const void *arg)
  */
 static inline int interrupt_get_irq(unsigned int irq, const char *cascade)
 {
-#if defined(CONFIG_LIBRARY) || defined(CONFIG_ACE) || defined(CONFIG_ZEPHYR_POSIX)
+#if defined(CONFIG_LIBRARY) || defined(CONFIG_ACE) || \
+	defined(CONFIG_ZEPHYR_POSIX) || CONFIG_ARM64
```

Comment:
Lacks consistency. Probably should do defined(CONFIG_ARM64) if there's no reason not to do that.